policy-controller/src/main.rs

```diff
@@ -0,0 +1,148 @@
+#![deny(warnings, rust_2018_idioms)]
+#![forbid(unsafe_code)]
+
+use anyhow::{Context, Result};
+use futures::{future, prelude::*};
+use linkerd_policy_controller::k8s::DefaultAllow;
+use linkerd_policy_controller_core::IpNet;
+use std::net::SocketAddr;
+use structopt::StructOpt;
+use tokio::{sync::watch, time};
+use tracing::{debug, info, instrument};
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "policy", about = "A policy resource prototype")]
+struct Args {
+    #[structopt(short, long, default_value = "0.0.0.0:8080")]
+    admin_addr: SocketAddr,
+
+    #[structopt(short, long, default_value = "0.0.0.0:8090")]
+    grpc_addr: SocketAddr,
+
+    /// Network CIDRs of pod IPs.
+    ///
+    /// The default includes all private networks.
+    #[structopt(
+        long,
+        default_value = "10.0.0.0/8,100.64.0.0/10,172.16.0.0/12,192.168.0.0/16"
+    )]
+    cluster_networks: IpNets,
+
+    #[structopt(long, default_value = "cluster.local")]
+    identity_domain: String,
+
+    #[structopt(long, default_value = "all-unauthenticated")]
+    default_allow: DefaultAllow,
+}
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    tracing_subscriber::fmt::init();
```

Comment:
we should probably turn off ANSI color codes — either here or via feature flags

Reply:
already off via feature flags